### ISSUE TITLE
sort players properly

### ISSUE THREAD

**Issue by jschnei** [ripgarpr/garpr#100]
In the list of players for a region (e.g. https://www.notgarpr.com/#/newjersey/players) players are sorted case-sensitively; we should instead sort them case-insensitively (i.e. sort by player_name.lower()).

**Comment by BrandonCookeDev** [comment]
Seems the sorting happens deep in the backend using Pymongo. As for mongo, this is apparently a known issue of Mongo not supporting non-case sensitive sorting.
http://stackoverflow.com/questions/19855147/mongodb-returns-capitalized-strings-first-when-sorting

However, we could either solve this with sorting the results in js or mongo aggregation (harder but more efficient):
http://stackoverflow.com/questions/19855147/mongodb-returns-capitalized-strings-first-when-sorting

**Comment by jschnei** [comment]
no need for anything fancy, just sort the return_dict in PlayerListResource. should be one line of code.

**Comment by BrandonCookeDev** [comment]
Alright, I'll just leave the above aggregation there in case we want to revisit. Probably is a more efficient way of doing this.

**Comment by BrandonCookeDev** [comment]
You were right, it was a quick sorted lambda query. Hotfix incoming.

```
return_dict['players'] = [p.get_json_dict() for p in sorted(all_players, key=lambda player : player.name.lower())]
```

**Comment by jschnei** [comment]
Closed by 420cf026f79dc8eac9b494f30ab8d3ca3dd5b92a
